.github/workflows/release-trello-actions.yml

```diff
@@ -0,0 +1,25 @@
+name: "Perform actions in Trello following release"
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  create-trello-list:
+    runs-on: ubuntu-latest
+
+    environment: trello
+
+    steps:
+      - name: Create list
+        run: |
+          list=$( curl --request POST --url "https://api.trello.com/1/lists?name=Done%20-%20${{ github.event.release.name }}&idBoard=${{ secrets.TEAM_GRAPHQL_BOARD }}&key=${{ secrets.API_KEY }}&token=${{ secrets.API_TOKEN }}&pos=bottom" | jq .id )
```

Comment:
question (non-blocking): Wdyt of appending `TRELLO_` to the secret keys/names, aka `TRELLO_API_KEY` and `TRELLO_API_TOKEN`? if one looks at the secrets not noticing (or knowing) that the environment is `trello` then it's hard to figure out what for instance `API_KEY` stands for.

Reply:
Good shout, done!